mlflow/entities/span.py

```diff
@@ -50,7 +50,7 @@ class SpanType:
 
 
 def create_mlflow_span(
-    otel_span: Any, trace_id: str, span_type: Optional[str] = None
+    otel_span: Any, trace_id: str, span_type: str | None = None
 ) -> Union["Span", "LiveSpan", "NoOpSpan"]:
```

Comment:
I noticed there are still some Union left but converting this to `X | Y` throws an error. `from __future__ import annotations` or Python >= 3.11 is required.